public/src/js/widgets/columns/packages.js

```diff
@@ -178,6 +179,7 @@ export default class Package extends ColumnWidget {
     }
 
     displaySearchResults() {
+        mediator.emit('package:searched');
```

Comment:
that's not where the event should be emitted. This is a getter, it's only used to understand whether results are visible. The event should be emitted in the callback of search.
That would allow to test both success and failures.

You can't rely on knockout calling this function at the right time.